docs/developers/celestia-node.mdx

```diff
@@ -333,7 +333,7 @@ rm -rf ~./celestia-light && celestia light init
 Your output will look similar to below:
 
 ```sh
-@ ~ % rm -rf ./celestia-light-arabica-1 && celestia light init
+@ ~ % rm -rf ./celestia-light && celestia light init
```

Comment:
delete this line too, same point as the above.

Reply:
✅